CSharpMath.Forms.Tests/TestHelpers/ComplexClassData.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+namespace CSharpMath.Forms.Tests.TestHelpers {
+  public abstract class ComplexClassData<T> : IEnumerable<object[]> where T : class {
```

Comment:
This is just duplicating TheoryData of Xunit.

Reply:
O, nice. I will delete this class and replace usings of it.